.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+exclude: '^$'
+fail_fast: true
+repos:
+-   repo: https://github.com/pre-commit/mirrors-yapf
```

Comment:
This does not seem to be v. actively used and maintained, what do you think about enabling without mirror?

Reply:
> This does not seem to be v. actively used and maintained

I am curious how did you decide this?

I see `https://github.com/pre-commit/mirrors-yapf` is updated whenever `yapf`(https://github.com/google/yapf/releases) is updated as you can see here https://github.com/pre-commit/mirrors-yapf/releases 

> what do you think about enabling without mirror
- We need to provide git repository to enable pre-commit hooks (Ref: http://pre-commit.com/#plugins)
- So What you mean by without mirror?

Reply:
> > This does not seem to be v. actively used and maintained

> I am curious how did you decide this?

> I see https://github.com/pre-commit/mirrors-yapf is updated whenever yapf(https://github.com/google/yapf/releases) is updated as you can see here https://github.com/pre-commit/mirrors-yapf/releases

Thanks for linking this, my observation was 👎 skipped over `53 releases`. 👍 

> > what do you think about enabling without mirror

> We need to provide git repository to enable pre-commit hooks (Ref: http://pre-commit.com/#plugins)
o w/ the number of releases they had, don't need this.

> So What you mean by without mirror?
☝️ what you understood, a fork of our own or a code to enable a pre-commit hook.